okhttp/src/main/kotlin/okhttp3/internal/Util.kt

```diff
@@ -378,18 +375,18 @@ fun Socket.peerName(): String {
  * @param source the source used to read bytes from the socket.
  */
 fun Socket.isHealthy(source: BufferedSource): Boolean {
-  try {
+  return try {
```

Comment:
For what it's worth, the return value here is non-obvious with the finally clause.  IntelliJ is obnoxious by making recommendations that make the code less clear.

Reply:
The specific reason I like the return here is that it forces a return expression from all branches of the clause.  Compiler would fail here anyway, but makes it clear that the result of the try catch is the final result of the method.

Reply:
Can I argue this one?

Reply:
Of course. And your argument makes sense.